MapzenSDKTests/MapViewControllerTests.swift

```diff
@@ -629,7 +627,7 @@ class MapViewControllerTests: XCTestCase {
     var loaded = false
     let styleLoaded : MZMapViewController.OnStyleLoaded = { (style) in loaded = true }
     try? controller.loadStyleAsync(.bubbleWrap, onStyleLoaded: styleLoaded)
-    controller.mapView(controller.tgViewController, didLoadSceneAsync: "/test/container/bubble-wrap/bubble-wrap-style-more-labels.yaml")
+    controller.mapView(controller.tgViewController, didLoadScene: 0, withError: nil)
```

Comment:
Should this be `didLoadSceneAsync` ?

Reply:
Nada - the callback from tangram-es is now `didLoadScene`.